core/internal/cltest/cltest.go

```diff
@@ -40,6 +40,9 @@ import (
 	"github.com/tidwall/gjson"
 	"github.com/urfave/cli"
 
+	"github.com/smartcontractkit/chainlink-relay/pkg/loop"
+	solana2 "github.com/smartcontractkit/chainlink-solana/pkg/solana"
```

Comment:
solanaPlugin rather than solana2?

Reply:
Eh well this package isn't plugin related, but IIRC we use `pkgsolana` other places